### ISSUE TITLE
Typescript documentation doesn't work

### ISSUE THREAD

**Issue by miguelchicoespin** [uktrade/cypress-image-diff#141]
I followed steps described in [Cypress integration](https://github.com/uktrade/cypress-image-diff/blob/main/docs/Cypress%20integration.md) but it doesn't work

![image](https://user-images.githubusercontent.com/39924906/218972979-f0f356d0-9022-404c-9aa8-bd62f420b8ef.png)

Ignoring that error and starting Cypress give me:

<img width="836" alt="image" src="https://user-images.githubusercontent.com/39924906/218973464-33151375-3a40-4396-8d71-c977e1a6fcb2.png">

**Comment by PippoRaimondiDIT** [comment]
Have you installed the package?

```
npm i -D cypress-image-diff-js
```

**Comment by dventsis-catalogicsoftware** [comment]
I have the  same issue with TypeScript
![image](https://user-images.githubusercontent.com/124582134/223679885-bb18f3be-0f39-4de4-9eda-e377e6e4a498.png)
![image](https://user-images.githubusercontent.com/124582134/223679954-1bc6ba4e-c753-4ad4-be9f-3d6ea0e3abb4.png)

**Comment by ger-benjamin** [comment]
The doc should says something like "add that in your (cypress) tsconfig.json file":
```
"types": [
  cypress-image-diff-js"
],
```
However, I currently can't make it work, same error.

**Comment by keith1976smith** [comment]
I'm hitting the same problem. Is there any chance this can be looked at please? It either works with typescript or it doesn't. Thank you!

**Comment by BaDo2001** [comment]
I know its not ideal but here is how I made it work for now:

__cypress/types.d.ts__

```typescript
declare module 'cypress-image-diff-js/dist/plugin' {
  const getCompareSnapshotsPlugin: (
    on: Cypress.PluginEvents,
    config: Cypress.PluginConfigOption,
  ) => void;

  export default getCompareSnapshotsPlugin;
}

declare module 'cypress-image-diff-js/dist/command' {
  const compareSnapshotCommand: () => void;

  export default compareSnapshotCommand;
}
```

__cypress/support/command.ts__

```typescript
/// <reference types="cypress" />
import type { RecurseDefaults } from 'cypress-recurse/src/index';

declare global {
  namespace Cypress {
    interface Chainable {
      compareSnapshot(
        name: string,
        testThreshold?: number,
        retryOptions?: Partial<typeof RecurseDefaults>,
      ): Chainable<Element>;
    }
  }
}
```

__cypress/tsconfig.json__
```json
{
  ...
  "include": ["**/*.ts", "types.d.ts"]
}
```

I am unsure why I can't move my type declarations from command.ts to types.d.ts as well but whatever, since I already have other commands defined there.

**Comment by samvienne** [comment]
Any news on this problem? Tks

**Comment by Maximillion195** [comment]
Try using a require instead of an import

`const getCompareSnapshotsPlugin = require('cypress-image-diff-js/dist/plugin');`

**Comment by raphaelbadia** [comment]
> I know its not ideal but here is how I made it work for now:
> 
> **cypress/types.d.ts**
> 
> ```ts
> declare module 'cypress-image-diff-js/dist/plugin' {
>   const getCompareSnapshotsPlugin: (
>     on: Cypress.PluginEvents,
>     config: Cypress.PluginConfigOption,
>   ) => void;
> 
>   export default getCompareSnapshotsPlugin;
> }
> 
> declare module 'cypress-image-diff-js/dist/command' {
>   const compareSnapshotCommand: () => void;
> 
>   export default compareSnapshotCommand;
> }
> ```
> 
> **cypress/support/command.ts**
> 
> ```ts
> /// <reference types="cypress" />
> import type { RecurseDefaults } from 'cypress-recurse/src/index';
> 
> declare global {
>   namespace Cypress {
>     interface Chainable {
>       compareSnapshot(
>         name: string,
>         testThreshold?: number,
>         retryOptions?: Partial<typeof RecurseDefaults>,
>       ): Chainable<Element>;
>     }
>   }
> }
> ```
> 
> **cypress/tsconfig.json**
> 
> ```json
> {
>   ...
>   "include": ["**/*.ts", "types.d.ts"]
> }
> ```
> 
> I am unsure why I can't move my type declarations from command.ts to types.d.ts as well but whatever, since I already have other commands defined there.

Thank you, works perfectly for me

**Comment by tamasmagyar** [comment]
Until v2.0.0 is released, can you try the [following Typescript guide](https://cypress.visual-image-diff.dev/getting-started/cypress-integration/typescript)?

**Comment by kien-ht** [comment]
This issue should be fixed after this pull request #195. Please upgrade to the latest version and see if the error still persists. 
I have created some [integration examples here](https://github.com/kien-ht/cypress-image-diff-html-report/tree/1aad688d5f4806be82a85c8bce1461cb0dbe4d79/examples) in case you want a reference.
